addons/modules/functions/fnc_moduleHideTerrainObjects.sqf

```diff
@@ -0,0 +1,55 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to hide terrain objects.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleHideTerrainObjects
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _position = ASLToAGL getPosASL _logic;
+deleteVehicle _logic;
+
+["STR_a3_to_hideTerrainObjects1", [
+    ["TOOLBOX:YESNO", "STR_a3_to_hideTerrainObjects1", true],
```

Comment:
It is a bit unclear that no means to unhide, would be nice to see the options be unhide/hide instead of no/yes.

Reply:
I made it the same way as Hide Zeus. I'll implement Hide/Unhide.

Reply:
5a42a2d